src/coinflip.rs

```diff
@@ -30,7 +30,22 @@ impl Side {
     }
 }
 
-/// Takes a randomness and returns the result of a coinflip (heads or tails)
+/// Takes a randomness and returns the result of a coinflip (heads or tails).
+///
+/// ## Example
+///
+/// ```
+/// use nois::coinflip;
+///
+/// # let randomness: [u8; 32] = [0x77; 32];
+/// let side = coinflip(randomness);
+/// if side.is_heads(){
+///     println!("heads")
+/// }
+/// if side.is_tails(){
+///     println!("tails")
+/// }
```

Comment:
This would be a more idiomatic way of working with the enum:

```suggestion
/// println!("Result: {side}");
/// match self {
///     Side::Heads => {
///         // Player A starts the game
///     },
///     Side::Tails => {
///         // Player B starts the game
///     },
/// }
```